README.md

```diff
@@ -1,7 +1,15 @@
 # InterSystems® Server Manager
-This is a VS Code helper extension that contributes settings which define connections to [InterSystems](https://www.intersystems.com/) servers.
+
+This is a VS Code helper extension that helps configure connections to [InterSystems](https://www.intersystems.com/) servers in VS Code extensions such as the [ObjectScript Extension for VS Code](https://github.com/intersystems-community/vscode-objectscript).
+
+It also enables the secure storage of passwords using the underlying operating system's native key store.
```

Comment:
```suggestion
It also enables the secure storage of passwords using the underlying operating system's native keystore.
```

Reply:
I wrote keystore in my DC post, and Jeff Fried changed it to 2 words, so I agree with you!